### PR TITLE
Settings, Version with About in Nav drawer with improvements

### LINKED ISSUE #396: Version and About section together in Nav drawer
**Actual Behaviour.**  
The two options are present at different locations.

**Expected Behaviour**  
Version and About section together in Nav drawer

**Steps to reproduce it**  
Check the navigation drawer

**Additional context**  
> Not available

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #396 

**Changes**: Settings, Version with About in Nav drawer with improvements

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
https://drive.google.com/open?id=1UtOvn1PiFrYS9NvPZDRGyF0QtBWj_Iss

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -130,7 +130,7 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="sleep_desc">Meditations while having trouble in sleeping</string>
     <string name="depressed_desc">Meditations when you feel depressed</string>
     <string name="happiness_desc">Meditations while you are in a happy mood</string>
-    <string name="focus_game_one_info">The Rocket Moves Up and Down according to your brain activity. Relax to travel forward or concentrate to travel back.</string>
+    <string name="focus_game_one_info">The rocket moves up and down according to your brain activity. Concentrate to fly up or relax to fly down.</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml:133]: I saw this in #395 Why don't you properly align branches before making PRs? I've told this many time.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (jddeep) [reply app/src/main/res/values/strings.xml]: Ah! Actually both the PRs happened to be from the sane branch. I was checking for these things only for which didn't request for a review :sweat_smile: . Will update.

Comment (jddeep) [reply app/src/main/res/values/strings.xml:133]: @CloudyPadmal This change from #395 didn't affect the codebase https://github.com/fossasia/neurolab-android/blob/development/app/src/main/res/values/strings.xml

Comment (jddeep) [reply app/src/main/res/values/strings.xml:133]: As a result of which this appears to be a change in this PR.

Comment (CloudyPadmal) [reply app/src/main/res/values/strings.xml:133]: That's strange. Why didn't that change make it through? :thinking:

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**